pallets/gear-debug/src/lib.rs

```diff
@@ -288,19 +288,20 @@ pub mod pallet {
         /// Used in tests to exhaust block resources.
         ///
         /// Parameters:
-        /// - `_fraction`: the fraction of the `max_extrinsic` the extrinsic will use.
+        /// - `fraction`: the fraction of the `max_extrinsic` the extrinsic will use.
         #[pallet::call_index(1)]
         #[pallet::weight({
             if let Some(max) = T::BlockWeights::get().get(DispatchClass::Normal).max_extrinsic {
-                *_fraction * max
+                *fraction * max
             } else {
                 Weight::zero()
             }
         })]
         pub fn exhaust_block_resources(
             origin: OriginFor<T>,
-            _fraction: Percent,
+            fraction: Percent,
         ) -> DispatchResultWithPostInfo {
+            let _ = fraction; // We dont need to check the weight witness.
```

Comment:
Even that I don't completely understand the meaning of this, still:

"We **dont** need" -> "We **don't** need" ?

Reply:
unused params in weights were deprecated